src/net_processing.cpp

```diff
@@ -422,7 +423,7 @@ class PeerManagerImpl final : public PeerManager
     std::atomic<int> m_best_height{-1};
 
     /** Next time to check for stale tip */
-    int64_t m_stale_tip_check_time{0};
+    std::chrono::seconds m_stale_tip_check_time{0s};
```

Comment:
While using `std::literals::chrono_literals` why not using `auto` with universal initialization:
```suggestion
    auto m_stale_tip_check_time{0s};
```

It looks concise and expressive, no?
Also could be applied in other places above.

Reply:
That’s a good suggestion. I have used the auto keyword at all the places where it could be used. See 92082ea0bb62ab6ec723b57ca4f2b2c5f6b28469
However, I could not use `auto` with the particular example you had taken. Doing so would cause the following error:

```
error: non-static data member declared with placeholder ‘auto’
```

The rationale for this behavior is discussed [here](https://stackoverflow.com/a/11303175).